app/templates/_base_page.html

```diff
@@ -18,6 +18,14 @@
     {% endblock %}
     <main id="content" role="main">
       {% include "toolkit/flash_messages.html" %}
+      {% block errorSummary %}
+        {% if errors %}
+          {{ govukErrorSummary({
+          "titleText": "There is a problem",
```

Comment:
Interesting that 'There is a problem' tested well with users (according to the [Design System issue](https://github.com/alphagov/govuk-design-system-backlog/issues/46)! It feels a little unnatural to me. I wonder if we'll run into scenarios in other apps where we need to overwrite it with something else...

Reply:
We've put the error summary into a block so it can be overridden for whatever reason; however I would like to think that this will be needed only rarely.

In general I think we should be moving our error messages to be based to Design System recommendations wherever possible, to reduce the amount of content we have to manage in our service.

Reply:
I think it would be good to get @NoraGDS to review and see if we can update the content. I have read some guidance somewhere on in the design system about trying not to blame the user. For example the existing error messages make me feel like I've done something terrible and it comes across as forceful ("You must..."). Design System do have patterns regarding error messages for specific things, for example, email address error messages (https://design-system.service.gov.uk/patterns/email-addresses/#error-messages)

Reply:
@aliuk2012 yes, the plan is to remove the "you must" messages 👍

Reply:
Yes, that would be my recommendation too. So if the 4 'You must provide' are removed and replaced with 'Enter', that would comply with the Design System too.